mgmtfn/dockplugin/netDriver.go

```diff
@@ -371,6 +371,58 @@ func leave(w http.ResponseWriter, r *http.Request) {
 	w.Write(content)
 }
 
+func networkAllocate(w http.ResponseWriter, r *http.Request) {
+	resp := api.AllocateNetworkResponse{}
+
+	content, err := json.Marshal(resp)
+	if err != nil {
+		httpError(w, "Could not generate a response", err)
+		return
+	}
+
+	w.Write(content)
```

Comment:
Eventhough we dont do any processing of these APIs, we need to return a valid response to avoid failure in dockerd. Will remove the return